projects/storefrontapp-e2e-cypress/cypress/helpers/coupons/my-coupons.ts

```diff
@@ -227,3 +289,24 @@ export function testClaimCustomerCoupon() {
     });
   });
 }
+
+export function testClaimCustomerCouponWithCodeInBody() {
+  describe('Claim customer coupon with code in requestBody', () => {
+    //TODO uncomment when enable 'enableClaimCustomerCouponWithCodeInRequestBody' to make ClaimCustomerCouponWithCodeInBody work in the backend, the new Occ endpoint is available since Commerce 2211.28.
+    it.skip('should claim customer coupon successfully with code in requestBody', () => {
```

Comment:
hi @rmch91 ,  skip new added cases for now after disable toggle, not sure when and who should enable the cases?
Thanks.

Reply:
I Respond in other comment